.github/dependabot.yml

```diff
@@ -35,8 +35,8 @@ updates:
 
   - package-ecosystem: "docker"
     directories:
-      - "/.docker/ubuntu-20.04"
       - "/.docker/ubuntu-22.04"
+      - "/.docker/ubuntu-24.04"
```

Comment:
> The directories key supports globbing and the wildcard character *. These features are not supported by the directory key.

https://docs.github.com/en/code-security/dependabot/working-with-dependabot/dependabot-options-reference#directories-or-directory--

Can we prevent this in the future by just doing `- "/.docker/*`?